src/materials/LineBasicMaterial.js

```diff
@@ -33,6 +33,7 @@ THREE.LineBasicMaterial = function ( parameters ) {
 	this.linejoin = 'round';
 
 	this.vertexColors = THREE.NoColors;
+	this.skinning = false;
```

Comment:
Nice. You added skinning support to `LineBasicMaterial`. The docs -- and the header inline comments --- need to be updated, too.